projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.ts

```diff
@@ -118,10 +149,54 @@ export class TableWidgetRendererComponent
     );
   }
 
+  public onFilterChange(item: ToggleItem<TableWidgetFilterModel>): void {
+    if (item.value && item.value.attribute && item.value.operator && item.value.value) {
```

Comment:
Is there a chance that these would be unset? What if one of the toggles was meant to clear the filters? Also, the value could be a falsy value like `0` or `false`, right?

Reply:
Fixed the 0/false case. Thanks.

We do have a use case for "All" asset tags that clears the filter. This works because `value` is `undefined` for that filter, therefore the `if` check fails and an empty array is provided thus clearing the filter.